share/spice/timer/timer.css

```diff
@@ -1,233 +1,321 @@
-.zci--timer .timer__btn {
-    height: 75px;
-    width: 75px;
-    border-radius: 37.5px;
-    text-align: center;
-    line-height: 75px;
-    border: 1px solid black;
-    font-size: 13px;
-    font-weight: bold;
-    margin-left: 5px;
-    margin-right: 5px;
-    outline: none; /*remove the blue outline in chrome/opera*/
-    background: none;
-    cursor: pointer;
+/* GENERAL */
+
+.zci--timer .cw {
+    max-width: 90em;
 }
 
-/*remove the dashed line in FF/IE*/
-.zci--timer .timer__btn::-moz-focus-inner {
-    border: 0;
+.zci--timer .zci__main {
+    max-width: 100%;
 }
-    
-.zci--timer .timer__btn.timer__start {
-    border-color: #20b84c;
-    background-color: #20b84c;
-    color: #fff;
+
+.zci--timer .zci__body {
+    padding-left: 0;
 }
 
-.zci--timer .timer__btn.timer__pause {
-    border-color: #ff3908;
-    background-color: #ff3908;
-    color: #fff;
+.zci--timer .timer_container {
+    position: relative;
+    width: 305px;
+    height: 140px;
+    display: block;
 }
 
-.zci--timer .timer__btn[disabled] {
-    color: #b4b4b4;
-    border-color: #b4b4b4;
-    cursor: default;
+.zci--timer .play_pause,
+.zci--timer .timer_container,
+.zci--timer #add_timer_container,
+.zci--timer .display_container {
+    float: left;
 }
 
-.zci--timer .timer__btn.timer__start[disabled] {
-    background-color: #ccc;
-    border-color: #ccc;
-    color: #f2f2f2;
+.zci--timer #add_timer_container {
+    display: block;
+    width: 140px;
+    height: 140px;
+    background-color: #e6e6e6;
```

Comment:
I think we need to use our special `bg-clr--foo` and `tx-clr--foo` CSS classes so that colours work across themes:

![timer_at_duckduckgo](https://cloud.githubusercontent.com/assets/873785/9231383/f985a716-40f5-11e5-808a-c368bc972b67.png)